src/main/java/numPeopleCheck.java

```diff
@@ -0,0 +1,42 @@
+import java.util.Scanner;
+
+public class numPeopleCheck {// класс, который проверяет количество человек, на которых нужно разделить счет
+    int checkStart() {
+        System.out.println("Здравствуйте! На сколько человек необходимо разделить счёт?");
+        int billCount;
+        while(true) {
+            Scanner scanner = new Scanner(System.in);
+            if (scanner.hasNextInt()) {
+                billCount = scanner.nextInt();
+            } else {
+                System.out.println("Вы ввели не целое число. Пожалуйста, введите целое значение");
+                continue;
+            }
+            if (billCountCheck(billCount)) {
+                break;
+            }
+        }
+        return billCount;
+    }
+    boolean billCountCheck(int billCount) {
```

Comment:
🍏 Методы, используемые только внутри класса, лучше сделать приватными.
```suggestion
    private boolean billCountCheck(int billCount) {
```